app/src/main/java/xyz/whereuat/whereuat/LoginActivity.java

```diff
@@ -169,12 +176,38 @@ private void showRequestHideCreate() {
         }
     }
 
+    private void showButtonHideSpinner() {
+        mSpinner.setVisibility(View.GONE);
+        mCreateBtn.setVisibility(View.VISIBLE);
+    }
+
+    private void showSpinnerHideButton() {
+        mSpinner.setAlpha(0f);
+        mSpinner.setVisibility(View.VISIBLE);
+
```

Comment:
nit: remove this blank line and move the `setAlpha` below `setVisibility`. I think it makes it easier to see that we're setting the alpha to 0 because we need to animate it to 1. It's kind of all one operation.